### PR TITLE
HyperShift: add 4.14 job

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-critical-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-critical-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-critical-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

**Comment by LiangquanLi930** [comment]
@liangxia Could you help to review this PR? Thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -853,6 +853,33 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
+- as: aws-ipi-ovn-hypershift-guest-p1-critical-f7
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:856]: no longer need `pN` in the test name.

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml]: @liangxia Thanks for your reminder, could you help to review it again?

**Pull request merged**